pulpcore/pulpcore/app/apps.py

```diff
@@ -111,11 +105,11 @@ def import_viewsets(self):
             for objname in dir(self.viewsets_module):
                 obj = getattr(self.viewsets_module, objname)
                 try:
-                    # Any subclass of GenericNamedModelViewSet that isn't itself
-                    # GenericNamedModelViewSet or other base viewset
+                    # Any subclass of NamedModelViewSet that isn't itself
+                    # NamedModelViewSet or other base viewset
                     # gets registered in the named_viewsets registry.
                     if (obj not in base_viewsets and
```

Comment:
this line should be updated to if obj is not NamedModelViewSet and issubclass(obj, NamedModelViewSet)